components/02-elements/table/_table.scss

```diff
@@ -27,89 +27,109 @@ $table-radius: $border-radius;
             }
         }
     }
-        
+
+    thead {
+        tr {
+            padding: 0;
+            border-bottom: 0;
+
+            @include mq($screen-m) {
+                border-top-left-radius: $table-radius;
+                border-top-right-radius: $table-radius;
+            }
+        }
+
+        th {
+            &:first-child {
+                @include mq($screen-m) {
+                    border-top-left-radius: $table-radius;
+                }
+            }
+
+            &:last-child {
+                @include mq($screen-m) {
+                    border-top-right-radius: $table-radius;
+                }
+            }
+        }
+    }
+
     tr {
         display: block;
-        padding: $table-padding 0;
         border-bottom: 1px solid $gray-lighter;
+        padding: 10px 0;
 
         @include mq($screen-m) {
             padding: 0;
             border-bottom: 0;
             display: table-row;
         }
-
-        &:first-child {
-            border-top-left-radius: $table-radius;
-            border-top-right-radius: $table-radius;
-        }
-
-        &:last-child {
-            border-bottom-right-radius: $table-radius;
-            border-bottom-left-radius: $table-radius;
-        }
     }
 
     th {
-        padding: 5px $table-padding;
-        background: $white;
-        display: block;
-        text-align: left;
+        display: none;
 
         @include mq($screen-m) {
             padding: $table-padding;
             background: $gray-lightest;
             border-bottom: 1px solid $gray-lighter;
             border-right: 1px solid $gray-lighter;
             display: table-cell;
-
-            &:first-child {
-                border-top-left-radius: 0;
-                border-top-right-radius: 0;
-            }
-
-            &:last-child {
-                border-bottom-right-radius: 0;
-                border-bottom-left-radius: 0;
-            }
-        }
-
-        &:first-child {
-            border-top-left-radius: $table-radius;
         }
 
         &:last-child {
-            border-top-right-radius: $table-radius;
-            border-right: 0;
+            @include mq($screen-m) {
+                border-right: 0;
+            }
         }
     }
 
     td {
         padding: 5px $table-padding;
         background: $white;
-        display: block;
+        display: flex;
         text-align: left;
 
+        &:before {
+            content: attr(data-th);
+            font-weight: bold;
+            margin-right: 10px;
+            max-width: 25%;
+            display: inline-block;
```

Comment:
if you use flex for container, there is no need to use `inline-block` for display. Inline-block add sometimes some strange alignment, I'd change it for `display: block` and to container I'd add: `flex-flow: row nowrap;` to make sure it will be hold in one line. What do you think?

Reply:
changed to `display:block;`